src/app/geo/windowRef/window-ref.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
```

Comment:
I answered it last week, but I don't know where it is now... Anyway
"Referencing global browser objects like document or window directly from within your code is possible, but not encouraged and considered bad practice.

Especially since Angular 2 isn’t only designed to run within your browser, but also on mobile, the server or web workers where objects like window may not be available."

https://dzone.com/articles/angular-2-how-do-i-get-a-reference-to-the-window-o